forms/offer/transport/OfferTransportGoodsForm.tsx

```diff
@@ -58,7 +58,11 @@ export const OfferTransportGoodsForm = ({
       .required(t('error.availability.required'))
       .typeError(t('error.must.be.string')),
     availability_interval_from: yup.mixed().typeError(t('error.must.be.time')),
-    availability_interval_to: yup.mixed().typeError(t('error.must.be.time')),
+    availability_interval_to: yup.mixed().typeError(t('error.must.be.time'))
```

Comment:
1. this validation should be added in `OfferTransportPersonsForm.tsx` also
2. please extract this validator in `validators.ts` see `dateStringValidator` or `dateInTheFutureValidator`